### ISSUE TITLE
Examine re-enabling reposts on odysee homepage

### ISSUE THREAD

**Issue by tzarebczan** [OdyseeTeam/odysee-frontend#37]
@kauffj commented on [Thu Apr 15 2021](https://github.com/lbryio/lbry-desktop/issues/5894)

- Must not show duplicates
- Put test version up on dev server for evaluation by growth team

---

@kauffj commented on [Mon Apr 19 2021](https://github.com/lbryio/lbry-desktop/issues/5894#issuecomment-822727297)

this probably requires SDK changes to do

---

@kauffj commented on [Mon Apr 19 2021](https://github.com/lbryio/lbry-desktop/issues/5894#issuecomment-822786285)

@shyba says that if `limit_claims_per_channel` is passed that this already happens

---

@infinite-persistence commented on [Thu Apr 29 2021](https://github.com/lbryio/lbry-desktop/issues/5894#issuecomment-829181774)

- Seems like already working:
    - ![image](https://user-images.githubusercontent.com/64950861/116547620-b75ad700-a925-11eb-8067-61650e557b9e.png)
- Also double-checked the code.  We are setting `limit_claims_per_channel` for the homepages.

---

@kauffj commented on [Thu Apr 29 2021](https://github.com/lbryio/lbry-desktop/issues/5894#issuecomment-829272416)

it has to be done in a way that the same claim doesn't show up twice, even if it is reposted

if it's not feasible to do cleanly with existing APIs then it should be pushed to SDK team to modify APIs to support this

---

@kauffj commented on [Thu Apr 29 2021](https://github.com/lbryio/lbry-desktop/issues/5894#issuecomment-829272888)

also, I believe all homepage search queries for odysee.com need to be modified to allow reposts as a type, but this should only be done if repetition problem is addressed 

---

@kauffj commented on [Tue Jun 01 2021](https://github.com/lbryio/lbry-desktop/issues/5894#issuecomment-852231179)

@shyba please comment when this is workable (i.e. live on wallet servers)

---

@kauffj commented on [Mon Jun 07 2021](https://github.com/lbryio/lbry-desktop/issues/5894#issuecomment-856023667)

On hold pending #6180 

---

@tzarebczan commented on [Fri Jun 11 2021](https://github.com/lbryio/lbry-desktop/issues/5894#issuecomment-859894861)

This can be worked on currently, not blocked by 6180. The separate changes were reverted on SDK for now. 

The SDKs / hubs were updated last night and now have this param:

```           
 --remove_duplicates             : (bool) removes duplicated content from search by picking either the
                                                     original claim or the oldest matching repost
```

We probably want this enabled by default on all claim searches. 

---

@kauffj commented on [Mon Jul 26 2021](https://github.com/lbryio/lbry-desktop/issues/5894#issuecomment-886798967)

- Restore reposts as being allowed on homepage claim_searches
- Add `remove_duplicates` flag

**Comment by kauffj** [comment]
@tzarebczan this is now workable and should be a one-line change
